doc/guides/01-why_interactions_tldr.md

```diff
@@ -0,0 +1,27 @@
+# Why Should you use Interactions?
+## ActiveInteraction
```

Comment:
I see in other files you've gone with "###" instead of "##" - let's stick with three for consistency